monster-js/xiaoqingb/js/daguaishou.js

```diff
@@ -0,0 +1,115 @@
+// 初始血量
+var newblood=100;
```

Comment:
```suggestion
var initialHealth=100;
var health = initialHealth;
```
我觉得你的血量混淆单词混淆了我，其实你可以有两个血量的概念。
一个是当前血量： `health`，另一个是最大血量，或者说是初始血量 `initialHealth`，用于说你复活的时候会设置的值，你参考一下~